### PR TITLE
Add worksection

### PR DESCRIPTION
**Pull request by NasirSethe**
-Created a branch to work on.
- Added HTML markup for cards and about section.
- Added styling to the project to replicate the template.
- Corrected linter errors.
- Committed the significant changes in between.
- Pushed to the remote branch and created a pull request.

### CODE REVIEW

File: index.html
Comment (Kamba56) [inline index.html:1]: - [ ] Kindly use an indentation of two spaces in all your documents. THis is a convention and also best practice.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+                <p>A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li class="one">HTML</li>
+                    <li class="two">CSS</li>
+                    <li class="three">Javascript</li>
+                </ul>
+
+                <button>See project</button>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly add a hover and pressed state to all your buttons to match the Figma template. You can use the `:hover` and `:active` pseudoclasses for this.
image

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+}
+
+/* WORK SECTION */
+
+.work-section {
+  display: flex;
+  flex-direction: column;
+  padding: 7.125rem 1.5rem;
+  gap: 5.5rem;
+}
```
Comment (Kamba56) [inline styles.css]: - [x] As demanded by the project requirements, kindly use CSS grid to lay out the container of all your work cards. Grid and flexbox aid in the responsiveness of your application, it is important you learn how to use both.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+                <button class="seeproject">See project</button>
+
+            </div>
+        </div>
+
+
+    </section>
+
+    <article class="about-section">
```
Comment (Kamba56) [inline index.html]: - [x] Kindly use a `section` element to wrap the About section. This is best for HTML semantics.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @NasirSethe, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using flexbox to layout contents of your page.👏👏👏
- [x] Youn did a good job sticking to the design. You will be a great developer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (NasirSethe) [reply styles.css]: Noted with thanks @Kamba56. I will make the necessary changes 👌🏾

File: index.html (hunk 3 above)
Comment (NasirSethe) [reply index.html]: @Kamba56  Noted with thanks.

File: index.html (hunk 1 above)
Comment (NasirSethe) [reply index.html]: @ The visuals provided here are quite helpful. Much appreciated.

File: index.html
Comment (NasirSethe) [reply index.html:1]: @Kamba56 Your guidance and kind words are highly appreciated 💯
Kindly expound a little more on this please. Not entirely sure I get what the expected changes are. Is it only in the index.html or the styles.css file as well? And an indentation of two spaces in the entire document for all elements to shift to the right? My apologies for the many questions 🙏🏾

**Review by braincee — APPROVED** [review]
Hi @NasirSethe 👋 , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html
Comment (Kamba56) [reply index.html:1]: > @Kamba56 Your guidance and kind words are highly appreciated 100 Kindly expound a little more on this please. Not entirely sure I get what the expected changes are. Is it only in the index.html or the styles.css file as well? And an indentation of two spaces in the entire document for all elements to shift to the right? My apologies for the many questions 🙏🏾

I apologize for replying late. Hope you got it now. Your CSS file already has two space indentation.

Comment (NasirSethe) [reply index.html:1]: That's okay. Got what you meant @Kamba56

**Pull request merged**